lib/internal/Magento/Framework/Data/Form/Element/Date.php

```diff
@@ -90,6 +90,7 @@ public function setValue($value)
 
         try {
             $this->_value = new \DateTime($value, new \DateTimeZone($this->localeDate->getConfigTimezone()));
+            $this->getValue();
```

Comment:
You get the value but don't assign it to any property. 
I don't understand how this changes can fix anything. 
Can you provide more information, please?

Reply:
Sometimes $this->_value can be set with an invalid value but so it will not work correctly. If $this->getValue(); runs then it effectively checks the value was set to correct value.

The time I experienced this was when a single letter string was given to  new \DateTime() as the value. It would set it but cause a fatal error when $this->getValue(); was run.

Reply:
VladimirZaets  how do I sign the cla?

Reply:
I have signed the cla